launchable/commands/record/commit.py

```diff
@@ -74,16 +74,32 @@ def commit(ctx, source: str, executable: bool, max_days: int, scrub_pii: bool, i
         )
         client.print_exception_and_recover(e)
 
+    cwd = os.path.abspath(source)
     try:
```

Comment:
I'm still wondering and It might be overkill.. But should we separate try-catch for exec_jar and checking shallow, just in case?

```python
try:
  is_shallow = ...
except Exception as e
  tracking_clinet.send_event(event_name=Tracking.Event.SHALLOW_CLONE, 'Cannot exec `git rev-parse --is-shallow-repository`, e)

try:
  exec.jar(...)
except Exception as e:
  click.echo(...)
  tracking_client.send_event(event_name=Tracking.Event.COLLECT_COMMIT)
```

What do you think??

Reply:
[247012c](https://github.com/launchableinc/cli/pull/900/commits/247012cbb1d1309f110733f350dc35c1219c9984)